modules/arrow/src/workers/triangulation-worker.ts

```diff
@@ -35,5 +44,35 @@ function triangulateBatch(data: TriangulateInput): TriangulateResult {
     data.flatCoordinateArray,
     data.nDim
   );
+  // @ts-expect-error
   return {...data, triangleIndices};
 }
+
+/**
+ * Reading the arrow file into memory is very fast. Parsing the geoarrow column is slow, and blocking the main thread.
+ * To address this issue, we can move the parsing job from main thread to parallel web workers.
+ * Each web worker will parse the geoarrow column using one chunk/batch of arrow data, and return binary geometries to main thread.
+ * The app on the main thread will render the binary geometries and the parsing will not block the main thread.
+ *
+ * @param data
+ * @returns
+ */
+function parseGeoArrowBatch(data: ParseGeoArrowInput): ParseGeoArrowResult {
+  let binaryGeometries: BinaryDataFromGeoArrow | null = null;
+  const {arrowData, chunkIndex, geometryColumnName, geometryEncoding, meanCenter, triangle} = data;
+  const arrowTable = arrow.tableFromIPC(arrowData);
```

Comment:
The raw Arrow arraybuffer  is zero copied to each web worker

Reply:
Not sure where this `arrowData` is coming from, but if you call `tableToIPC` somewhere, you're making a copy.